linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/tests/pam_tally2_auth_unlock_time_configured.pass.sh

```diff
@@ -4,15 +4,28 @@
 cat >/etc/pam.d/common-account <<CAPTC
 account	[success=1 new_authtok_reqd=done default=ignore]	pam_unix.so
 account	requisite			pam_deny.so
-account required                        pam_tally2.so
+account required            pam_tally2.so
 account	required			pam_permit.so
 CAPTC
 
-cat >/etc/pam.d/common-account <<CAPTUTC
+{{% if product in ["sle12","sle15"] %}}
+
+cat >/etc/pam.d/login <<CAPTUTC
```

Comment:
The only change between this jinja2 conditional is the destination file. You can include only the destination file in the conditional and avoid many repeated lines.

Reply:
Actually, the rule is restricted to `prodtype: sle12,sle15`. So, the conditional is not necessary. You can simply update the file. Valid for the other comments too.